qlib/data/ops.py

```diff
@@ -1541,6 +1626,7 @@ def _load_internal(self, instrument, start_index, end_index, *args):
 
 TOpsList = [TResample]
 OpsList = [
+    ChangeInstrument,
```

Comment:
I'm not sure if I'm correct.
Personally, it confuses me a little when I see the word change.
Do you think "FixedInst" will be a better name?

Reply:
Maybe SwitchInstrument or ReplaceInstrument is a better name? Although it's unlikely to have multiple chained ChangeInstrument op, but in principle one can do that. FixedInst implies one can't do that.

Reply:
OK.
I think `ChangeInstrument` will be OK now.